lib/Sema/CSGen.cpp

```diff
@@ -1286,6 +1257,19 @@ namespace {
       return witnessType;
     }
 
+    Type visitRegexLiteralExpr(RegexLiteralExpr *E) {
+      auto &ctx = CS.getASTContext();
+      auto *regexDecl = ctx.getRegexDecl();
```

Comment:
What if the _StringProcessing module failed to load? We have a warning in the frontend, but should we also bail here with an error to avoid crashing?

Reply:
Sure, done.  Though Concurrency doesn't check this (e.g. call site of `ASTContext::getUnsafeContinuationDecl()`).